<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MngFormCategory</class>
 <widget class="QDialog" name="MngFormCategory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manutenção de Categorias</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/PNG/128x128/gnome-system-config-128x128.png</normaloff>:/images/PNG/128x128/gnome-system-config-128x128.png</iconset>
  </property>
  <widget class="QGroupBox" name="GBoxViewCategory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>731</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Categorias</string>
   </property>
   <widget class="QTableView" name="TableViewCategory">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>591</width>
      <height>261</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Categorias Disponíveis...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonRemoveCategory">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Fazer a Remoção do Item Selecionado da Tabela...</string>
    </property>
    <property name="text">
     <string>Remover</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/delete-48x48.png</normaloff>:/images/PNG/48x48/delete-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonCleanCategory">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Limpar os Campos de Cadastro e Atualizar a Janela de Manutenção...</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/Rebuild-48x48.png</normaloff>:/images/PNG/48x48/Rebuild-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GBoxDataCategory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>161</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="title">
    <string>Dados</string>
   </property>
   <widget class="QLabel" name="LabelCodeCategory">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Código:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelNameCategory">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelShortNameCategory">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome Curto:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonSaveCategory">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Salvar os dados para a Categoria...</string>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/Filesave-48x48.png</normaloff>:/images/PNG/48x48/Filesave-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="CBoxCodeCategory">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar um Código para a Categoria...</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditIDCategory">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditNameCategory">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar uma  Descrição/Nome para a Categoria...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditShortNameCategory">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar um Nome Curto/Sigla/Apelido para a Categoria...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="GBoxCategoryStatusActive">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>171</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QCheckBox" name="CheckBoxCategoryStatusActive">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Permite Ativar/Desativar a Categoria para ser ou não Usada...</string>
     </property>
     <property name="text">
      <string>Ativo</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolButton" name="TButtonExitCategory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sair da Janela de Manutenção...</string>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/PNG/48x48/Exithome-48x48.png</normaloff>:/images/PNG/48x48/Exithome-48x48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
